content/docs/guides/contribute/conventions/back.en.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Back-end"
+linkTitle: "Back-end"
+description: "Coding style guide and best practices for back-end"
+---
+
+# Python
+
+Python code is used for some packages and integration testing.
+
+- Follow [the Zen of Python](https://www.python.org/dev/peps/pep-0020/).
+- Code is linted with [flake8](https://github.com/csachs/pyproject-flake8).
+- Code is formatted with [Black](https://github.com/psf/black).
+- Imports are sorted with [Isort](https://github.com/PyCQA/isort).
+- Python tests are written using [pytest](https://docs.pytest.org/).
```

Comment:
Didn't we get rid of python now that api is dead ?

Reply:
I've not updated this text, you should ask dev mainteners :slightly_smiling_face:

Reply:
Resolve is at your discretion

Reply:
I asked @flomonster, there is still some python so we should keep this